phpunit/LDAP/AuthLdapTest.php

```diff
@@ -374,39 +374,39 @@ public function testBuildLdapFilter()
         /** @var \AuthLDAP $ldap */
         $ldap = getItemByTypeName('AuthLDAP', 'LDAP3');
         $result = \AuthLDAP::buildLdapFilter($ldap);
-        $this->assertSame("(& (email=*) )", $result);
+        $this->assertSame("(&(email=*) )", $result);
```

Comment:
Should we adapt the code to change this too ?

```suggestion
        $this->assertSame("(&(email=*))", $result);
```